### DISCUSSION TITLE
Serverpod does not have "generated" folder

### DISCUSSION THREAD

**Discussion by letscodetpl** [serverpod/serverpod#2335] [Q&A] [answered]
Hi, I have a problem when creating serverpod project. It does not have "generated" folder outside of lib under the "deploy folder" (it does have a generated folder inside lib/src folder). Really appreciate if you can point out what is the issue. Thanks a lot.
![image](https://github.com/serverpod/serverpod/assets/171565733/e82e4d98-0bc9-4a74-b64e-a775c4e2819e)

**Comment by Isakdl** [comment]
The generated folder in the root was deprecated / removed in 1.2 I believe . What are you looking for?

**Reply by letscodetpl** [reply]
I am looking for the queries generated for database table. I am going through the tutorial on serverpod documentation but cannot find it.

**Reply by Isakdl** [reply, marked as answer]
I see, this is now handled with the migration system released in 1.2. 

You can find the tables in the `migrations` folder. For how to create and apply migrations see the docs: https://docs.serverpod.dev/concepts/database/migrations

**Reply by letscodetpl** [reply]
thank you so much
